plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -0,0 +1,686 @@
+local Blitbuffer = require("ffi/blitbuffer")
+local CenterContainer = require("ui/widget/container/centercontainer")
+local Device = require("device")
+local DocSettings = require("docsettings")
+local Font = require("ui/font")
+local FrameContainer = require("ui/widget/container/framecontainer")
+local Geom = require("ui/geometry")
+local GestureRange = require("ui/gesturerange")
+local HorizontalGroup = require("ui/widget/horizontalgroup")
+local HorizontalSpan = require("ui/widget/horizontalspan")
+local ImageWidget = require("ui/widget/imagewidget")
+local InfoMessage = require("ui/widget/infomessage")
+local InputContainer = require("ui/widget/container/inputcontainer")
+local LeftContainer = require("ui/widget/container/leftcontainer")
+local LineWidget = require("ui/widget/linewidget")
+local OverlapGroup = require("ui/widget/overlapgroup")
+local RightContainer = require("ui/widget/container/rightcontainer")
+local TextBoxWidget = require("ui/widget/textboxwidget")
+local TextWidget = require("ui/widget/textwidget")
+local UIManager = require("ui/uimanager")
+local UnderlineContainer = require("ui/widget/container/underlinecontainer")
+local VerticalGroup = require("ui/widget/verticalgroup")
+local VerticalSpan = require("ui/widget/verticalspan")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local lfs = require("libs/libkoreader-lfs")
+local util = require("util")
+local _ = require("gettext")
+local Screen = Device.screen
+local T = require("ffi/util").template
+
+local BookInfoManager = require("bookinfomanager")
+
+-- Here is the specific UI implementation for "list" display modes
+-- (see covermenu.lua for the generic code)
+
+
+-- We will show a rotated dogear at bottom right corner of cover widget for
+-- opened files (the dogear will make it look like a "used book")
+local corner_mark = ImageWidget:new{
+    file = "resources/icons/dogear.png",
+    rotation_angle = 270
+}
+
+
+-- ItemShortCutIcon (for keyboard navigation) is private to menu.lua and can't be accessed,
+-- so we need to redefine it
+local ItemShortCutIcon = WidgetContainer:new{
+    dimen = Geom:new{ w = 22, h = 22 },
+    key = nil,
+    bordersize = 2,
+    radius = 0,
+    style = "square",
+}
+
+function ItemShortCutIcon:init()
+    if not self.key then
+        return
+    end
+    local radius = 0
+    local background = Blitbuffer.COLOR_WHITE
+    if self.style == "rounded_corner" then
+        radius = math.floor(self.width/2)
+    elseif self.style == "grey_square" then
+        background = Blitbuffer.gray(0.2)
+    end
+    local sc_face
+    if self.key:len() > 1 then
+        sc_face = Font:getFace("ffont", 14)
```

Comment:
(That too is original menu.lua code, not mine.)
For the other coming font definitions you're going to see, I initially started by using picks from these set of standard fonts. Had to adjust some for the layout, and went back to using the standard "cfont" with the size that made each look fine, and fit in the hardcoded scaleBySize(64).